### ISSUE TITLE
Cannot disable creation of `ginkgo.report` files

### ISSUE THREAD

**Issue by millyleadley** [joselitofilho/ginkgoTestExplorer#48]
When I click "Run Test" it runs the test but also spews out a `ginkgo.report` file which I have to delete. Is there a way to disable this pretty please?

**Comment by joselitofilho** [comment]
@millyleadley,

No, there is not. I will create a task to remove the `ginkgo.report` after the execution.
